src/javascript/app/pages/user/set_currency.js

```diff
@@ -276,13 +276,27 @@ const SetCurrency = (() => {
                 }
             });
         } else {
-            if ($submit) {
-                $submit.removeClass('button-disabled');
-            }
+            removeError(null, true);
             $error.text(localize('Please choose a currency')).setVisibility(1);
         }
     };
 
+    /**
+     * Remove error text if $error is defined
+     * Enable confirm button if isBtnEnable is true
+     *
+     * @param {object} $error // error text jquery element
+     * @param {boolean} isBtnEnable // Enable button
+     */
+    const removeError = ($error, isBtnEnable)=>{
```

Comment:
for variables, we separate them with underscores:
```suggestion
    const removeError = ($error, is_btn_enabled) => {
```